### ISSUE TITLE
pod search can't find version 2.0.4

### ISSUE THREAD

**Issue by Anonymous-Monk** [airbnb/lottie-ios#383]
## pod search only find version：1.5.2，
-> lottie-ios (1.5.2)
   Used to natively render vector animations exported from After Effects.
   pod 'lottie-ios', '~> 1.5.2'
   - Homepage: https://github.com/airbnb/lottie-ios
   - Source:   https://github.com/airbnb/lottie-ios.git
   - Versions: 1.5.2, 1.5.1, 1.5.0, 1.0.4, 1.0.2, 1.0.1, 1.0.0 [master repo]

-> lottie-ios-extension (1.5.1.1)
   extension of Lottie-iOS for getting some properties to customize
   LOTAnimationView
   pod 'lottie-ios-extension', '~> 1.5.1.1'
   - Homepage: https://github.com/jegumhon/lottie-ios
   - Source:   https://github.com/jegumhon/lottie-ios.git
   - Versions: 1.5.1.1, 1.5.1 [master repo]

**Comment by buba447** [comment]
Results when I run `pod search lottie-ios`

```

-> lottie-ios (2.0.6)
   Used to natively render vector animations exported from After Effects.
   pod 'lottie-ios', '~> 2.0.6'
   - Homepage: https://github.com/airbnb/lottie-ios
   - Source:   https://github.com/airbnb/lottie-ios.git
   - Versions: 2.0.6, 2.0.5, 2.0.4, 2.0.3, 2.0.2, 2.0.1, 2.0.0, 1.5.2, 1.5.1, 1.5.0, 1.0.4, 1.0.2, 1.0.1, 1.0.0 [master repo]

-> lottie-ios-extension (1.5.1.1)
   extension of Lottie-iOS for getting some properties to customize LOTAnimationView
   pod 'lottie-ios-extension', '~> 1.5.1.1'
   - Homepage: https://github.com/jegumhon/lottie-ios
   - Source:   https://github.com/jegumhon/lottie-ios.git
   - Versions: 1.5.1.1, 1.5.1 [master repo]
(END)
```

I think your local cache is out of date.
This is a cocoapod issue, not a lottie issue.
Try running `pod cache clean` or `rm -rf ~/Library/Caches/CocoaPods`

**Comment by Anonymous-Monk** [comment]
Thanks,it's my problem!




发自网易邮箱大师
On 08/24/2017 05:50, Brandon Withrow wrote:

Closed #383.
